### DISCUSSION TITLE
HandleEventAsync  Calling interfaces that require identity authentication

### DISCUSSION THREAD

**Discussion by bieshaoxiong** [abpframework/abp#19480] [Q&A]
Hi，
In my message subscription, I need to call the interface that requires identity authentication, and now I keep getting an error message: Exception of type 'Volo.' Abb Authorization AbpAuthorizationException 'was thrown May I ask how to solve this problem now
![image](https://github.com/abpframework/abp/assets/64937534/befe2184-bc34-4faa-a1f7-eab1a3fee6d1)
![image](https://github.com/abpframework/abp/assets/64937534/c3aa3f7d-a2b2-4605-8c89-02cfbf4fba28)

**Comment by maliming** [comment]
hi

You can use Domain Service in your EventHandler.
